### PR TITLE
Ensure terminology

### PR DESCRIPTION
**Pull request by GMartinez-Sisti**
- Scripts tested
- Read everything twice (but do check)

### CODE REVIEW

File: console/configurations/add_proxy.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 - **Host**: IP/DNS record used to reach the proxy
-- **Port**: Port where the proxy will be available
+- **Port**: Port where the Fyde Access Proxy will be available
 
 Click **Create**.
 
 ![Add Proxy](imgs/access_add_proxy.png)
 
-Copy the link that will be used when configuring the proxy.
+Copy the enrollmennt link that will be used when configuring the Fyde Access Proxy.
```
Comment (oNaiPs) [inline console/configurations/add_proxy.md]: Typo enrollmennt

File: glossary.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Envoy Proxy is part of the Fyde Access Proxy combo, it listens to requests and proxies them to the correct destination.
+
+## Fyde Access Proxy
+
+Fyde Access Proxy is the software combo that contains Envoy Proxy and Fyde Proxy Orchestrator.
+
+## Fyde Access Proxy enrollment link
+
+Required by Fyde Access Proxy and used tto bootstrap and authorize the service. Obtained only when created a Fyde Access Proxy.
```
Comment (oNaiPs) [inline glossary.md]: typo to

File: console/configurations/add_proxy.md (hunk 1 above)
Comment (GMartinez-Sisti) [reply console/configurations/add_proxy.md]: Fixed

File: glossary.md (hunk 2 above)
Comment (GMartinez-Sisti) [reply glossary.md]: Fixed

File: proxy/docker/docker-compose.yml [hunk 3]
```diff
@@ -4,23 +4,28 @@ services:
...
     ports:
-      # Publicly exposed Access Proxy port must match in the management console
+      # Publicly exposed Fyde Access Proxy port
+      # Must match the one define in Fyde Enterprise Console
       - "8000:8000"
     restart: unless-stopped
 
-  proxy-client:
-    container_name: proxy-client
+  fyde-proxy:
```
[the remark this answers is not in the document]
Comment (GMartinez-Sisti) [reply proxy/docker/docker-compose.yml]: Alright I'll change it.

File: fyde_access_proxy.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Fyde Access Proxy
```
[the remark this answers is not in the document]
Comment (GMartinez-Sisti) [reply fyde_access_proxy.md]: Ok.

**Comment by GMartinez-Sisti** [comment]
Suggestions applied. Please approve again.

**Pull request merged**